spec.bs

```diff
@@ -447,6 +448,83 @@ To <dfn>build an interest group permissions url</dfn> given a [=origin=] |ownerO
 
 </div>
 
+<h3 id="interest-group-storage-maintenance">Interest Group Storage Maintenance</h3>
+
+There is a job that periodically [=performs storage maintenance=] on the user agent's
+[=interest group set=]. It performs operations such as [=list/removing=] expired or excess
+[=interest groups=]. It's required that [=interest group set=] has no more than:
+  * 2000 [=interest group/owners=];
+  * 1000 [=regular interest groups=] per [=interest group/owner=];
+  * 20000 [=negative interest groups=] per [=interest group/owner=];
+  * <code>10\*1024\*1024</code> total [=interest group/estimated size=] of [=interest groups=] per
+    [=interest group/owner=].
+
+<div algorithm='maintenance'>
+To <dfn>perform storage maintenance</dfn>:
+
+1. Let |ownersAndExpiry| be a new [=ordered map=] whose [=map/keys=] are [=origins=]
+  [=interest group/owner=] and [=map/values=] are [=moments=] [=interest group/expiry=].

```

Comment:
I think `[=interest group/expiry=]` isn't needed since the type is `[=moment=]` which is defined and already linked to. We could add a note that in reality, the moments stored in this new ordered map will always be from an IG/expiry though. That would probably be nice.